convex/chatsHttp.ts

```
@@ -0,0 +1,198 @@
+import { httpAction } from './_generated/server';
+import { internal } from './_generated/api';
+import { createClient } from '@supabase/supabase-js';
+
+// Initialize Supabase client
+const supabase = createClient(
+  process.env.NEXT_PUBLIC_SUPABASE_URL!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)